benches/helpers.rs

```diff
@@ -32,7 +31,7 @@ pub async fn ws_server() -> String {
 		let server = WsServerBuilder::default().build("127.0.0.1:0").await.unwrap();
 		let mut module = RpcModule::new(());
 		module.register_method(SYNC_METHOD_NAME, |_, _| Ok("lo")).unwrap();
-		module.register_async_method(ASYNC_METHOD_NAME, |_, _| (async { Ok("lo") }).boxed()).unwrap();
+		module.register_async_method(ASYNC_METHOD_NAME, |_, _| (async { Ok("lo") })).unwrap();
```

Comment:
Do we still need the parens around the `async` block?

Reply:
Right, don't think they were needed there to begin with!

Reply:
I thought that too when I saw that they weren't used below in a couple of places :D